operator/avs_manager.go

```diff
@@ -285,6 +286,21 @@ func (avsManager *AvsManager) RegisterOperatorWithAvs(
 	return nil
 }
 
+func (avsManager *AvsManager) DeregisterOperator(blsKeyPair *bls.KeyPair,) (error) {
```

Comment:
```suggestion
func (avsManager *AvsManager) DeregisterOperator(blsKeyPair *bls.KeyPair) error {
```

Reply:
Surprising that `gofmt` does not format this automatically